pkg/microservice/aslan/core/common/repository/models/service.go

```diff
@@ -78,6 +78,7 @@ type Service struct {
 	EnvName            string                           `bson:"env_name,omitempty"             json:"env_name,omitempty"`
 	TemplateID         string                           `bson:"template_id,omitempty"          json:"template_id,omitempty"`
 	AutoSync           bool                             `bson:"auto_sync"                      json:"auto_sync"`
+	Production         bool                             // check current service data is production service
```

Comment:
no, this field just used to check this service data whether is test service or production service, because I reused a method ensureServiceTmpl() of the original service.